### PR TITLE
Fix/update dependencies

### PR DESCRIPTION
**Pull request by lhotanok**

### CODE REVIEW

File: src/enqueue_places.js [hunk 1]
```diff
@@ -293,12 +297,14 @@ module.exports.enqueueAllPlaceDetails = async ({
     try {
         await page.click('#searchbox-searchbutton');
     } catch (e) {
-        log.warning(`click#searchbox-searchbutton ${e?.message}`);
+        const error = /** @type {Error} */ (e);
+        log.warning(`click#searchbox-searchbutton ${error.message}`);
         try {
             const retryClickSearchButton = await page.$('#searchbox-searchbutton');
-            await retryClickSearchButton.evaluate(b => b.click());
+            await retryClickSearchButton?.evaluate(b => b.click());
```
Comment (pocesar) [inline src/enqueue_places.js:304]: this should fail so the `catch` block can be reached for the fallback submit. but a more explicit `throw` here might be better

File: src/main.js [hunk 2]
```diff
@@ -202,6 +202,27 @@ Apify.main(async () => {
 
     const proxyConfiguration = await Apify.createProxyConfiguration(proxyConfig);
 
+    /**
+     * `stealthOptions` property is not declared in `launchContext`.
+     * It is passed to Puppeteer launcher through `launchContext`:
+     * `const puppeteerLauncher = new PuppeteerLauncher(launchContext);`
+     * `puppeteerLauncher.stealthOptions`
+     */
+     const stealthOptionsWrapper = {
```
Comment (pocesar) [inline src/main.js:211]: since stealth is deprecated, I think we can get rid of it altogether

**Review by pocesar — CHANGES_REQUESTED** [review]
looks very good, thanks for also updating the comments and docs @lhotanok after merge, going to do `beta` on platform

File: src/main.js (hunk 2 above)
Comment (lhotanok) [reply src/main.js:211]: Yeah, I was thinking the same but wasn't sure

**Review by pocesar — APPROVED** [review]

**Review by metalwarrior665 — APPROVED** [review]
Looks good but we should test more as there are a lot of small changes

**Pull request merged**